### PR TITLE
Kerlscan changes to run Drift Backend in Clowder

### PR DESCRIPTION
**Pull request by jpramos123**
## Secure Coding Practices Checklist Link
- https://github.com/RedHatInsights/secure-coding-checklist

## Secure Coding Checklist
- [ ] Input Validation
- [ ] Output Encoding
- [ ] Authentication and Password Management
- [ ] Session Management
- [ ] Access Control
- [ ] Cryptographic Practices
- [ ] Error Handling and Logging
- [ ] Data Protection
- [ ] Communication Security
- [ ] System Configuration
- [ ] Database Security
- [ ] File Management
- [ ] Memory Management
- [x] General Coding Practices

### CODE REVIEW

File: kerlescan/config.py [hunk 1]
```diff
@@ -13,12 +14,37 @@ def str_to_bool(s):
...
+
+baseline_svc_hostname = load_hosts_setting(
+    "BASELINE_SVC_URL", "baseline", "http://baseline_svc_url_is_not_set"
+)
+
+rbac_svc_hostname = load_hosts_setting("RBAC_SVC_URL", "rbac", "http://rbac_svc_url_is_not_set")
+
+hsp_svc_hostname = load_hosts_setting("HSP_SVC_URL", "hsp", "http://hsp_svc_url_is_not_set")
+
+drift_svc_hostname = load_hosts_setting("DRIFT_SVC_URL", "drift", "http://drift_svc_url_is_not_set")
```
Comment (fellipeh) [inline kerlescan/config.py:45]: Will be great if we move forward with constants in uppercase: https://www.python.org/dev/peps/pep-0008/#constants
```suggestion
INVENTORY_SVC_HOSTNAME = load_hosts_setting(
    "INVENTORY_SVC_URL", "host-inventory", "http://inventory_svc_url_is_not_set"
)

BASELINE_SVC_HOSTNAME = load_hosts_setting(
    "BASELINE_SVC_URL", "baseline", "http://baseline_svc_url_is_not_set"
)

RBAC_SVC_HOSTNAME = load_hosts_setting("RBAC_SVC_URL", "rbac", "http://rbac_svc_url_is_not_set")

HSP_SVC_HOSTNAME = load_hosts_setting("HSP_SVC_URL", "hsp", "http://hsp_svc_url_is_not_set")

DRIFT_SVC_HOSTNAME = load_hosts_setting("DRIFT_SVC_URL", "drift", "http://drift_svc_url_is_not_set")
```

**Review by fellipeh — CHANGES_REQUESTED** [review]
Nice work. Just small comments/request changes

**Review by fellipeh — CHANGES_REQUESTED** [review]
Please rebase into 1 commit

File: kerlescan/config.py (hunk 1 above)
Comment (fellipeh) [reply kerlescan/config.py:45]: I don't know the reason of github duplicate my requested changes. Resolving this.

**Review by CodeHeeler — COMMENTED** [review]
Looks good but I am not knowledgeable in Clowder changes; maybe also ping someone from outside the team who has been working on Clowder with you to glance over the PR as well.  Thank you for doing this work!

File: kerlescan/config.py [hunk 2]
```diff
@@ -12,13 +12,32 @@ def str_to_bool(s):
...
+
+inventory_svc_hostname = load_hosts_setting("INVENTORY_SVC_URL", 'host-inventory', "http://inventory_svc_url_is_not_set")
+
+baseline_svc_hostname = load_hosts_setting("BASELINE_SVC_URL", 'baseline', "http://baseline_svc_url_is_not_set")
+
+rbac_svc_hostname = load_hosts_setting("RBAC_SVC_URL", 'rbac', "http://rbac_svc_url_is_not_set")
+
+hsp_svc_hostname = load_hosts_setting("HSP_SVC_URL", 'hsp', "http://hsp_svc_url_is_not_set")
+
+drift_svc_hostname = load_hosts_setting("DRIFT_SVC_URL", 'drift', "http://drift_svc_url_is_not_set")
```
[the remark this answers is not in the document]
Comment (jpramos123) [reply kerlescan/config.py]: @fellipeh Understood, but can we hold on this changes for now?

As we have 3 services dependent on this repo, we will need to apply those changes there as well... 

Any thoughts?

[the remark this answers is not in the document]
Comment (fellipeh) [reply kerlescan/config.py]: Sure.. fair enough... no need to change it on the others services for now.

**Review by fellipeh — APPROVED** [review]
LGTM

File: Pipfile [hunk 3]
```diff
@@ -21,6 +21,7 @@ boto3 = "*"
 watchtower = "*"
 insights-core = "*"
 bitmath = "*"
+app-common-python = {git = "https://github.com/RedHatInsights/app-common-python"}
```
Comment (petrblaho) [inline Pipfile:24]: Do we really want to get that code from master all the time?

We don't have direct control over it so I suggest to do it by git tags.

Maybe we can ask how often `app-common-python` is tagged? (@psav?)

**Review by petrblaho — CHANGES_REQUESTED** [review]
LGTM but I would be happy if we would solve the dep version of `app-common-python`

File: Pipfile (hunk 3 above)
Comment (fellipeh) [reply Pipfile:24]: @petrblaho I asked @jpramos123 about that, and he told me `app-common-python` doesn't use tag for while..  the last tag as in April, and there's no tag anymore...

Comment (jpramos123) [reply Pipfile]: @fellipeh @petrblaho In a conversation with @psav he told me that we should use `pip install app-common-python` as it is in pip.

**Review by petrblaho — APPROVED** [review]
LGTM!

**Pull request merged**